portfolio/src/main/webapp/script/comment-script.js

```diff
@@ -12,12 +12,30 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/** @const commentHyphen used to space out comment text from author name */
```

Comment:
I believe this comment can be made into a summary. So you can just write: Spaces out comment text from author name.

Reply:
Done